scripts/consolidate-hyperindex-docs.js

```
@@ -0,0 +1,422 @@
+    }
+    // Skip other types like "link"
+  }
+
+  return fileOrder;
+}
+
+// Function to parse HyperSync sidebar configuration
+function parseHyperSyncSidebarOrder(sidebarPath) {
+  try {
+    const sidebarContent = fs.readFileSync(sidebarPath, "utf8");
+    const sidebarConfig = eval(`(function() {
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)